x-pack/plugins/monitoring/public/components/alerts/alerts.js

```diff
@@ -169,4 +167,4 @@ const AlertsUI = ({ alerts, angular, sorting, pagination, onTableChange, intl })
   );
 };
 
-export const Alerts = injectI18n(AlertsUI);
+export const Alerts = AlertsUI;
```

Comment:
@maryia-lapata I was initially going to do this, but noticed some `*UI` variables were also exported eg: https://github.com/elastic/kibana/pull/37845/files#diff-4d75d804e32b80fb0822600f0ec07ed6R86 which means they might be used somewhere else without internationalization (maybe testing?). So, I decided to leave these refactors for when I'll (eventually) start porting everything to typescript.

I'll still remove the ref variables and change any affecting exports

Reply:
I think  `*UI` variables export was left just in case or by mistake, in test there is a way to get access to inner component: e.g. `ApmServerInstances.WrappedComponent`.

Reply:
^^ This is true. It's most likely inconsistent for no real reason